### DISCUSSION TITLE
PROJ Problems during SamGeo Setup for windows - Python 3.12, GDAL 3.8.4 (installed from .whl)

### DISCUSSION THREAD

**Discussion by ibrahimnzamane** [opengeos/segment-geospatial#324] [Q&A] [answered]
I am trying to setup samgeo on multiple windows machines using both conda and pip. I keep getting "PROJ proj.db contains DATABASE.LAYOUT.VERSION.MINOR = 2 whereas a number >=3 is expected. It comes from another PROJ installation".

I have tried reinstalling GDAL, rasterio etc but the error persists.
It shows up when I try to access the downloaded 'tms as tiff file' from local machine and add it to map using m.add_raster(image, layer_name).

Any suggestion/help will be highly appreciated.

**Comment by giswqs** [comment]
How did you install it? 

https://samgeo.gishub.org/installation/#install-from-conda-forge

**Reply by ibrahimnzamane** [reply]
I created venv, using pip I installed jupyterlab, segment-geospatial and ran the sam2_predictor notebook. It gave missing GDAL error after Installing GDAL using whl file and downgrading numpy to 1.26.*.  I ran the notebook and recieved the above error. I have tried several times surprisingly it actually ran a a couple of times but gave the same error on reloading the notebook.

**Reply by giswqs** [reply, marked as answer]
You should use conda install gdal. pip is not recommended. See https://samgeo.gishub.org/installation/#install-from-conda-forge

**Reply by ibrahimnzamane** [reply]
I followed the installation guideline for conda-forge and then I updated the segment-geospatial from 0.10.7 to 0.11.2 using pip -U install segment-geospatial; which worked fine without any erros. However, Sometimes the raster layer and satellite basemap layer occupies only the top left corner of the output map cell generated by leafmap. This seems to be related to ileaflet or leafmap and not samgeo or is it something that I might have missed? Thank You.
